front/components/assistant/conversation/WebsearchAction.tsx

```diff
@@ -1,4 +1,4 @@
-import type { WebsearchActionType } from "@dust-tt/types/dist/front/assistant/actions/websearch";
+import type { WebsearchActionType } from "@dust-tt/types/dist";
```

Comment:
This solo import looks fishy?

Reply:
the 'dist' is def a typo if that's what you meant :+1: 
the import is necessary to the signature of WebsearchAction (implemented in the following PR)

Reply:
Ah yeah didn't realize it was an edit. But yeah the edit looks fishy also :p